.travis.yml

```diff
@@ -40,10 +40,10 @@ jobs:
 #      script: mvn test
 #      after_success: mvn prepare-package -DskipTests=true
   allow_failures:
+    - os: osx
     - jdk: openjdk-ea
```

Comment:
Note to self - the early access versions aren't run, so this "allow failures" has no effect.